runtime/lua/vim/filetype.lua

```diff
@@ -1608,6 +1625,10 @@ local filename = {
   ['Gopkg.lock'] = 'toml',
   ['/.cargo/credentials'] = 'toml',
   ['Cargo.lock'] = 'toml',
+  ['.black'] = 'toml',
+  black = function(path, bufnr)
+    return M._getline(bufnr, 1) == "'tool.black'" and 'toml' or nil
```

Comment:
```suggestion
    return M._getline(bufnr, 1) == 'tool.black' and 'toml' or nil
```

Reply:
Don't know what I was thinking there... (But note "black" instead of upstream "back".)